digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/SpeedLimitService.scala

```diff
@@ -306,17 +331,17 @@ class SpeedLimitService(eventbus: DigiroadEventBus, vvhClient: VVHClient, roadLi
     * Saves speed limit values when speed limit is separated to two sides in UI. Used by Digiroad2Api /speedlimits/:speedLimitId/separate POST endpoint.
     */
   def separate(id: Long, valueTowardsDigitization: Int, valueAgainstDigitization: Int, username: String, municipalityValidation: Int => Unit): Seq[SpeedLimit] = {
-    val speedLimit = withDynTransaction { dao.getPersistedSpeedLimit(id) }
+    val speedLimit = withDynTransaction { dao.getPersistedSpeedLimit(id)
       .map(toSpeedLimit)
       .map(isSeparableValidation)
-      .get
+      .get }
 
```

Comment:
We don't need to extend"withDynTransaction" to all maps. Since we will not use the "toSpeedLimit" on the new update method.